knative-operator/cmd/manager/main.go

```diff
@@ -124,6 +125,16 @@ func main() {
 
 	log.Info("Starting the Cmd.")
 
+	go func() {
+		// This web server is unimportant enough to not bother connecting its lifecycle to
+		// signal handling so the process can just tear it down with it.
+		log.Info("Serving CLI artifacts on :8080")
+		http.Handle("/", http.FileServer(http.Dir("/cli-artifacts")))
+		if err := http.ListenAndServe(":8080", nil); err != nil && err != http.ErrServerClosed {
+			log.Error(err, "Failed to launch CLI artifact server")
+		}
+	}()
+
```

Comment:
Do we have an easy way to E2E test the download ? I think it should not be that hard to add this to the test, just download some or all of the binaries.

Reply:
We have that (though just doing HEAD requests and making sure the binary size is plausible)

Reply:
See https://github.com/openshift-knative/serverless-operator/blob/main/test/servinge2e/verify_deploy_resources_test.go for posterity